modules/react/popup/lib/Popper.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import * as PopperJS from '@popperjs/core';
 
-export type Placement = PopperJS.Placement;
+export type Placement = `${PopperJS.Placement}`; // Use template literals to make documentation list them out
```

Comment:
wat

this is ok I'm just confused why this works

Reply:
This is a Template literal type. It tells Typescript to map out the string union. Otherwise the type would be `PopperJS.Placement` as an exported symbol. But our parser doesn't run on files inside `node_modules`, so this would be an empty symbol. In order to prevent this, I put in a template literal to force Typescript (and by extension the doc parser) to treat `PopperJS.Placement` as a _type_ and not a _symbol_.